### PR TITLE
admin: Fixed SPICE reconnection

### PR DESCRIPTION
**Pull request by olivergs**

### CODE REVIEW

File: admin/js/profiles.livesession.js [hunk 1]
```diff
@@ -18,72 +18,105 @@
...
+      $('#spinner-modal').modal('hide');
+      if (connecting) {
+        clearTimeout(connecting);
+        connecting = null;
+      }
+      showMessageDialog('Connection error to virtual machine.', 'Connection error');
     }
-  }
 
-  function agent_connected(sc) {
```
Comment (aruiz) [inline admin/js/profiles.livesession.js]: Why did you move this function?

File: admin/js/profiles.livesession.js [hunk 2]
```diff
@@ -18,72 +18,105 @@
...
-      onerror: spice_error
+      // onagent: agent_connected,
     });
   }
 
   try {
     do_connection();
   } catch (e) {
-    console.error('Fatal error:' + e.toString());
+    console.error('FC: Fatal error:' + e.toString());
```
Comment (aruiz) [inline admin/js/profiles.livesession.js]: Why is this necessary?

Comment (olivergs) [inline admin/js/profiles.livesession.js]: Because in console log, there are a lot of messages with a similar structure like Error, Fatal Error etc. And Sometimes I'm not able to know in a look if the message is something we are writing or is the SPICE client.

For that I prepended all messages using FC:

File: admin/js/profiles.livesession.js (hunk 1 above)
Comment (olivergs) [inline admin/js/profiles.livesession.js]: Probably moved it to the "end" of callback functions because nowadays it's not being used and my idea was to test to enable it again later because I changed some references that could be the reason of disconnections when we enabled it in the past.

Comment (aruiz) [inline admin/js/profiles.livesession.js]: Wouldn't it be better to test such change separately from adding this feature? At the very least in another commit with its own explanation

Comment (olivergs) [inline admin/js/profiles.livesession.js]: Ok. Will move it to where it was and do that in another commit and pull request

**Comment by aruiz** [comment]
looks good, merge
